File: __tests__/fixtures/output/csharp/httpclient/cookies.cs
var clientHandler = new HttpClientHandler
{
    UseCookies = false,
};
var client = new HttpClient(clientHandler);
var request = new HttpRequestMessage
{
    Method = HttpMethod.Get,
    RequestUri = new Uri("https://httpbin.org/cookies"),
    Headers =
    {
        { "cookie", "foo=bar; bar=baz" },
    },
};
using (var response = await client.SendAsync(request))
{
    response.EnsureSuccessStatusCode();
    var body = await response.Content.ReadAsStringAsync();
    Console.WriteLine(body);
}

File: __tests__/__fixtures__/output/node/request/cookies.js
const request = require('request');

const jar = request.jar();
jar.setCookie(request.cookie('foo=bar'), 'https://httpbin.org/cookies');
jar.setCookie(request.cookie('bar=baz'), 'https://httpbin.org/cookies');

const options = {method: 'GET', url: 'https://httpbin.org/cookies', jar: jar};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

